processor/elasticapmprocessor/README.md

```diff
@@ -0,0 +1,19 @@
+# Elastic APM connector
+
+The Elastic APM connector transforms and enriches signals with Elastic APM-specific
+attributes, and pre-aggregates Elastic APM-specific metrics.
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: traces, logs, metrics   |
+| Distributions | [] |
+| Warnings      | [Statefulness](#warnings) |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/elastic/opentelemetry-collector-components?query=is%3Aissue%20is%3Aopen%20label%3Aprocessor%2Felasticapm%20&label=open&color=orange&logo=opentelemetry)](https://github.com/elastic/opentelemetry-collector-components/issues?q=is%3Aopen+is%3Aissue+label%3Aprocessor%2Felasticapm) [![Closed issues](https://img.shields.io/github/issues-search/elastic/opentelemetry-collector-components?query=is%3Aissue%20is%3Aclosed%20label%3Aprocessor%2Felasticapm%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/elastic/opentelemetry-collector-components/issues?q=is%3Aclosed+is%3Aissue+label%3Aprocessor%2Felasticapm) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
+<!-- end autogenerated section -->
+
+## Configuration
+
+This connector currently has no configuration.
```

Comment:
Not true, it has configuration for the aggregation database directory and metadata keys